spacy/ml/parser_model.pyx

```diff
@@ -347,6 +347,7 @@ cdef class precompute_hiddens:
     cdef bint _is_synchronized
     cdef public object ops
     cdef public object numpy_ops
+    cdef public object cpu_ops
```

Comment:
Shall we underscore this one like for the `transition_parser`?

Reply:
Fixed.